### PR TITLE
ENG-1207 Create build pipeline for percona everest cli

### PR DESCRIPTION
**Pull request by adivinho**

### CODE REVIEW

File: packaging/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (michal-kralik) [inline packaging/LICENSE:1]: We have a `LICENSE` file in the root. Is there any reason we need a second license?

File: packaging/rpm/percona-everest-cli.spec [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%files
+%{_bindir}/everest
+%license LICENSE
+
+%changelog
+* Tue Aug 15 2023 Vadim Yalovets <vadim.yalovets> 0.1-1
```
Comment (michal-kralik) [inline packaging/rpm/percona-everest-cli.spec:57]: Our versioning consists of 3 versions `0.1.0`. Does the spec support this?

**Review by michal-kralik — COMMENTED** [review]
Thank you for your contribution.
Is the deployment meant to be run automatically on every new version?

File: packaging/LICENSE (hunk 1 above)
Comment (adivinho) [reply packaging/LICENSE]: It has been modified to use already provided LICENSE file

File: packaging/rpm/percona-everest-cli.spec (hunk 2 above)
Comment (adivinho) [reply packaging/rpm/percona-everest-cli.spec]: Fixed

**Comment by adivinho** [comment]
> Thank you for your contribution. Is the deployment meant to be run automatically on every new version?

There is no such task in the ticket.
Please create a new ticket in order to add an additional pipeline to run this job if a new tag is added.

**Review by michal-kralik — APPROVED** [review]

**Review by recharte — CHANGES_REQUESTED** [review]
Hi @adivinho thank you very much for this, it looks great.

I know that you asked for and got a sign-off on the binary file and package description. However, last week, when we released everest v0.1.0, we decided that the binary will be called `everestctl` and I guess this wasn't communicated to you and you used `everest` throughout.
May you please rename it to `everestctl`? Sorry for the extra work.

**Review by gen1us2k — APPROVED** [review]
Thanks. Looks good but please take a look at Diogo's request for changes.

**Comment by adivinho** [comment]
> Hi @adivinho thank you very much for this, it looks great.
> 
> I know that you asked for and got a sign-off on the binary file and package description. However, last week, when we released everest v0.1.0, we decided that the binary will be called `everestctl` and I guess this wasn't communicated to you and you used `everest` throughout. May you please rename it to `everestctl`? Sorry for the extra work.

It has been renamed.
e.g.
 
Screenshot 2023-08-29 at 10 09 31

**Review by recharte — APPROVED** [review]
Thank you, great job @adivinho

**Pull request merged**